docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
+
+To test highly available apiservers, you can modify local-up-cluster.sh as follows.
+
+1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
+
+2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
```

Comment:
This.  This this this.

I do not understand why a node-port is useful here?  In fact, why would this work at all?  I don't expect 127.0.0.1 to be a valid SAN on the certificate.

All node-port does is give you a different ingress into the service.  If the service properly contains multiple masters, the node port should be identical in plumbing to `kubernetes.local`.  Please explain?